app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -59,6 +61,17 @@ class EventDetailsFragment : Fragment() {
 
         eventViewModel.loadEvent(eventId)
 
+        //Set toolbar title to event name
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            rootView.nestedContentEventScroll.setOnScrollChangeListener { _, _, scrollY, _, _ ->
+                if (scrollY > rootView.eventName.height + rootView.logo.height) {
+                    activity?.supportActionBar?.title = eventShare.name
+                } else {
+                    activity?.supportActionBar?.title = " "
+                }
```

Comment:
What does this do?

Reply:
When we scroll downwards, after we have scrolled a the length of event name and image, event name will be shown. If we scroll upwards, event name disappears

Reply:
Write this in comment

Reply:
Added comment :+1: